daemon/api_snaps_test.go

```diff
@@ -1947,10 +1947,10 @@ func (s *snapsSuite) TestErrToResponseNoSnapsDoesNotPanic(c *check.C) {
 	}
 
 	for _, err := range errors {
-		rsp := si.ErrToResponse(err)
+		ae := si.ErrToResponse(err)
 		com := check.Commentf("%v", err)
-		c.Check(rsp, check.NotNil, com)
-		status := rsp.(*daemon.Resp).Status
+		c.Check(ae, check.NotNil, com)
```

Comment:
While you're at it, this should probably be changed to Assert, otherwise the next check may panic.